Client/src/Components/Sidebar/index.css

```diff
@@ -92,3 +92,7 @@ aside .MuiListSubheader-root {
     visibility: visible;
   }
 }
+
+.MuiTabPanel-root.css-8i83hh-MuiTabPanel-root .MuiStack-root.field.field-text.css-qm2p9n-MuiStack-root, .MuiTabPanel-root.css-2aha5r-MuiTabPanel-root .MuiStack-root.field.field-text.css-78xnin-MuiStack-root {
+  display: none;
+}
```

Comment:
This is in the sidebar CSS file but it is targeting a field in the password panel component.  This is unmaintainable as future devs will not know this CSS is in this file, nor will they know what it is for if they come across this CSS in the future.

The specificity of this CSS selector is also not really maintainable as it is not reasonably possible to tell what that selector is for.  I assume it was extracted from dev tools inspector?  It works, but no one is going to know what it is for or what it means :thinking: